extensions/amp-list/0.1/amp-list.css

```diff
@@ -14,6 +14,13 @@
  * limitations under the License.
  */
 
+/**
```

Comment:
Correct, this is by design since the current fallback/placeholder mechanism is insufficient for some UX patterns. Allowing the publisher to decide the treatment offers the most flexibility here.

Re: (3), we may need to add a resize request on `[fetch-error]` display. I'll add this when I split the PR.